pyproject.toml

```diff
@@ -1,48 +1,97 @@
 [build-system]
+requires = ["setuptools>=45",
+            "setuptools_scm[toml]>=6.2",
+]
+build-backend = 'setuptools.build_meta'
 
-requires = ["setuptools",
-            "setuptools_scm",
-            "wheel"]
+[project]
```

Comment:
Verified these changes with pip and this is the Before and After the PR:

**Before** 
![Screenshot from 2022-12-09 12-02-45](https://user-images.githubusercontent.com/19628176/206758041-20da7e28-363b-49e0-b56a-178be40991fe.png)

**After**
![pip_show_hermes_core](https://user-images.githubusercontent.com/19628176/206758009-7f560508-c552-4a75-9235-ce0b44323b16.png)

Should we include the Home-page and Author? It looks like it doesn't parse the name out automatically when defining `authors`